pagesnotfound.php

```diff
@@ -103,6 +103,8 @@ private function getPages()
 
     public function hookDisplayAdminStatsModules()
     {
+	$this->context->controller->addCSS($this->_path . 'views/css/stacking_responsive.css');
```

Comment:
Wouldn't it impact all the .table displayed in this hook, even tho you added a CSS in this single module? Maybe we should be explicit and have something like:

`.table.table-stacking`?

Reply:
Done. Explicit `.stacking__wrapper` is added before `.table`.